### DISCUSSION TITLE
Citation rendering not working

### DISCUSSION THREAD

**Discussion by danilomm** [jgm/pandoc#9721] [General]
Hi, 
I have been using citation rendering for over a decade on Mac OS (options ```-C``` or ```--citeproc```). 
Lately I have moved into Ubuntu 22.04.4 LTS. 
Pandoc has been working fine for months. But today I tried for the first time to use citation rendering, and it just returned: 
```Unknown option -C.```
```Try pandoc --help for more information.```
```shell returned 6```

The same with ```--citeproc```
Am I doing something wrong? Has it changed in someway that I did not reach?
Thanks in advance.

**Comment by jgm** [comment]
It's possible that Ubuntu has an old version of pandoc.
What does `pandoc --version` say?

**Reply by danilomm** [reply]
pandoc 2.9.2.1
Compiled with pandoc-types 1.20, texmath 0.12.0.2, skylighting 0.8.5

**Reply by jgm** [reply]
`--citeproc/-C` was added in 2.11.
You are using a very old version.  You can install the .deb package we provide on the Releases page to have a an up-to-date version.

**Comment by denismaier** [comment]
It should be this one:
https://packages.ubuntu.com/jammy/pandoc

Gesendet von Outlook für Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by denismaier** [comment]
By the way, Ubuntu 24.04 LTS has a newer version: https://packages.ubuntu.com/noble/pandoc

Gesendet von Outlook für Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by denismaier** [comment]
You should update: https://pandoc.org/releases.html#pandoc-2.11-2020-10-11

Gesendet von Outlook für Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by danilomm** [reply]
Thank you. I have updated and it worked fine! That's odd, since the ubuntu version is newer than 2020...
